crates/cxx-qt-lib/src/core/qtlogging.rs

```diff
@@ -36,6 +36,18 @@ mod ffi {
         /// Outputs a message in the Qt message handler.
         fn qt_message_output(msgType: QtMsgType, context: &QMessageLogContext, string: &QString);
 
+        /// Generates a formatted string out of the type, context, str arguments.
+        #[cxx_name = "qFormatLogMessage"]
+        fn format_log_message(
+            msgType: QtMsgType,
+            context: &QMessageLogContext,
+            string: &QString,
+        ) -> QString;
+
+        /// Generates a formatted string out of the type, context, str arguments.
```

Comment:
This doc comment belongs to qFormatLogMessage, not qSetMessagePattern.

Reply:
Fixed